driver_cpl/cime_config/buildnml

```diff
@@ -5,7 +5,7 @@
 # pylint: disable=invalid-name
 
 # Disable these because this is our standard setup
-# pylint: disable=wildcard-import,unused-wildcard-import,wrong-import-position
+# pylint: disable=wildcard-import,unused-wildcard-import
```

Comment:
which pylint is that?   Seems different pylint versions support different options and I'm at a bit of a loss regarding how to handle that.